### ISSUE TITLE
Fingerprint based on HTTP_ACCEPT Headers

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#2229]
**Describe the issue you encountered:** The language setting of OS/Brave seems to make it easy to detect uniqueness of the browser based on the HTTP_ACCEPT Headers.

**Expected behavior:** Is it possible to make the value of the language in the header undefined, unless doing so rather makes the browser more unique?
- Screenshot if needed: On panopticlick
  ![](https://cloud.githubusercontent.com/assets/3362943/16168341/cc24a31a-353e-11e6-91e2-8763a15000b4.jpg)
- Any related issues: #242

**Comment by diracdeltas** [comment]
i think your results are worse than most due to not many people in Japan visiting panopticlick.org. my Brave profile has `text/html, */*; q=0.01 gzip, deflate en-US`, which is 1 in 76. 

in Chrome: `text/html, */*; q=0.01 gzip, deflate, br en-US,en;q=0.8` is 1 in 13. not sure if there are any negative side effects from just copying the Chrome header.

**Comment by luixxiul** [comment]
I tested these two cases on chrome canary:

1. Enable both English and Japanese, and display Chrome in Japanese
2. Disable English, enable only Japanese and display Chrome in Japanese

In case 1, I got ``text/html, */*; q=0.01 gzip, deflate, br ja,en-US;q=0.8,en;q=0.6``, which is 1 in 3591.
In case 2, I got ``text/html, */*; q=0.01 gzip, deflate, br ja``, which is 1 in 20010.

I assume that even only including the string ``en`` would make the browser less unique.

**Comment by luixxiul** [comment]
On 0.10.4
![](https://cloud.githubusercontent.com/assets/3362943/16572172/8b6c91e2-42a0-11e6-9f1c-d54763e3719d.jpg)
It became worse than 0.10.3 where I got 1 in 10780

**Comment by luixxiul** [comment]
Was this fixed on 0.11.0? I get ``text/html, */*; q=0.01 gzip, deflate en`` though the language has been set to Japanese.

**Comment by luixxiul** [comment]
On macOS with 0.11.2 Beta1,
<img width="612" alt="screenshot 2016-07-29 21 28 40" src="https://cloud.githubusercontent.com/assets/3362943/17248163/7f6409de-55d3-11e6-902a-1613be6b1eb5.png">
````
text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8 gzip, deflate ja
````

Also the 16.81 bit of identifying information is largest among others.
